app/services/supported_ssg_updater.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+# This class updates supported SSGs yaml
+class SupportedSsgUpdater
+  require 'yaml'
+
+  KEYS_TO_REMOVE = %w[upstream_version brew_url policy_version _latest_policy_version _technical_confidence].freeze
```

Comment:
The last 3 keys should not be specified, everything under a profile should be just simply dropped. It would make testing easier.